src/Loaders/LoadingSpinner.tsx

```diff
@@ -0,0 +1,102 @@
+/** @jsx jsx */
+import React from "react";
+import { jsx, keyframes } from "@emotion/core";
+import { colors } from "../colors";
+
+export type Size = "large" | "medium" | "small" | "tiny" | "icon";
+export type Theme = "light" | "dark";
+interface Props {
+  /**
+   * Class name that will be applied to the svg
+   */
+  className?: string;
+
+  /**
+   * Theme for the spinner
+   */
+  theme?: Theme;
+
+  /**
+   * Size of the spinner
+   */
+  size?: Size;
```

Comment:
First, this isn't in the README, but it should be!

Can you add the default value here so when we have automatic prop table generation (hopefully in a few working days, this is a new priority along with other docs work), then the default will be appropriately listed?

Example: https://github.com/strothj/react-docgen-typescript-loader#exporting-components

Same for all other defaulted props please (meaning `theme`).

Reply:
Thats really cool, I didnt know about that!

Reply:
I didn't either :) None of the `Button`s props have the `@default` yet 🙈